<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInputsConfig</class>
 <widget class="QDialog" name="DialogInputsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка входов точки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Вход 1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxDI1">
        <property name="text">
         <string>Разрешение</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Фильтр, с:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxDI1Filter">
        <item>
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Группа:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBoxGroups"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Вход 2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxDI2">
        <property name="text">
         <string>Разрешение</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Фильтр, с:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxDI2Filter">
        <item>
         <property name="text">
          <string>0.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Точка:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBoxPoints"/>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Записать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogInputsConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogInputsConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
